chapters/extensions/VK_KHR_depth_stencil_resolve.md

```diff
@@ -0,0 +1,9 @@
+# VK_KHR_depth_stencil_resolve
+
+> Promoted to core in Vulkan 1.2
+
+This extension was created to support doing resolve on depth and stencil images. The main use case for this is tile-based architectures where doing MSAA on depth and stencil targets is expensive due to not having a way to keep the memory in the tiler memory.
```

Comment:
It's not the MSAA itself that is expensive, it's that prior to this extension there was no way to resolve the depth/stencil samples without writing them to memory first.
Maybe flip it the last sentence to something like "The main use case for this is tile-based architectures where it is more efficient to resolve the depth and stencil in tiler memory than to write unresolved samples to memory before doing the resolve in a separate step."

Reply:
ok, will do